tests/test_xtalimaging.py

```diff
@@ -50,5 +53,6 @@ def test_xtal_imaging(testdb):
         position1y=687,
         position2x=473,
         position2y=744,
+        experiment_type='MAD',
```

Comment:
I guess you're not using pre-commits, as I would have thought `black` would have changed this to double quotes?
```suggestion
        experiment_type="MAD",
```